testbed/HIL_Firmware/common/include/dac.h

```diff
@@ -0,0 +1,32 @@
+#ifndef DAC_H
+#define DAC_H
+
+#include "driver/spi_master.h"
+
+#define BYTE_0 0x02
+#define TX_LENGTH 24
+#define V_REF 3300.0
+#define MAX_STEPS_12 2048
```

Comment:
not really sure what this means without context. Macro names should be intuitive and almost read like a sentence